packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1030,6 +1034,9 @@ class EditableText extends StatefulWidget {
   /// {@end-tool}
   final ValueChanged<String> onSubmitted;
 
+  /// Called when receives the app private command.
```

Comment:
Called when the result of an app private command is received?

Reply:
Done.